html_app/ui/MainFrame.js

```diff
@@ -4,7 +4,7 @@ scb.ui = scb.ui || {};
 scb.ui.static = scb.ui.static || {};
 
 scb.ui.static.MainFrame = scb.ui.static.MainFrame || {};
-scb.ui.static.MainFrame.LOGOUT_URL = '/accounts/logout/';
+scb.ui.static.MainFrame.LOGOUT_URL = '/scb/logout/';
```

Comment:
How do I get to this? After I log in on the main page I still see 'Sign in'. When I click it it says that my account is confirmed